src/PlotlyJS.jl

```diff
@@ -4,6 +4,8 @@ using JSON
 using Blink
 using Colors
 
+import Base: print
```

Comment:
I'd prefer that we don't do the import and that in json.jl we just do `Base.print` to add a method